internal/packmgr/installer_test.go

```
@@ -0,0 +1,358 @@
+}
+
+func TestInstallRejectsInvalidSource(t *testing.T) {
+	ctx := context.Background()
+	root := t.TempDir()
+	cases := []struct {
+		name   string
+		mutate func(string)
+	}{
+		{name: "missing manifest", mutate: func(_ string) {}},
+		{name: "invalid manifest json", mutate: func(dir string) {
+			os.WriteFile(filepath.Join(dir, "manifest.json"), []byte("{"), 0o640)
```

golangci-lint (2.12.2)

[error] Error return value of `os.WriteFile` is not checked

(errcheck)